src/main/java/constants/Options.java

```diff
@@ -9,17 +9,26 @@ public class Options {
     public static final String MANGA_LONG_OPTION = "manga";
     public static final String MANGA_OPTION_DESC = "Manga name";
 
-    public static final String DEADLINE_DATE_OPTION = "d";
+    public static final String DEADLINE_DATE_OPTION = "b";
     public static final String DEADLINE_LONG_OPTION = "deadline";
     public static final String DEADLINE_OPTION_DESC = "Deadline date";
```

Comment:
After our offline discussion, perhaps changing these to something along the lines of "by date" or "by" would be better to signal to the user that this option is for a deadline without having to refer to the UG.